example/types/index.ts

```diff
@@ -18,6 +18,7 @@ export type ApiUpdate =
 export type ApiRequest =
   | Authorize
   | SumOfNumbers
+  | FileRequest
```

Comment:
```suggestion
  | UploadFile
```
cause it is the type of example of usage, it should look like in a real-world